tools/python/CMakeLists.txt

```diff
@@ -1,47 +1,12 @@
 
-CMAKE_MINIMUM_REQUIRED(VERSION 3.8.0)
-
-if (WIN32 AND NOT "${CMAKE_GENERATOR}" MATCHES "Visual Studio") 
-   message(FATAL_ERROR "\n"
-      "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!\n"
-      "You must use Visual Studio to build a python extension on windows. If you "
```

Comment:
Totally keep this.  People are so confused about how to build python extensions on windows.  Like massively confused.

Reply:
Like we need to keep all the warning messages like this. They all got added after I got a ton of such questions and eventually went and added some automatic warning to tell people the answer to their problems to preempt them from posting questions about it or emailing me.  Especially this one.  A ton of people installing python on windows don't know what visual studio is or that they have to install the C++ compiler part of it.  I still get questions about this and have to tell people to install it even with this warning being printed for them.

Reply:
Ok, but again, how much hand holding do we want to do?

Reply:
Maybe one of the windows runners should build the python extensions

Reply:
And dlib is all about having good docs and this is part of that.